types/index.esm.d.ts

```diff
@@ -1495,7 +1495,14 @@ export interface CoreChartOptions<TType extends ChartType> extends ParsingOption
   events: (
     'mousemove' |
     'mouseout' |
+    'mouseenter' |
+    'mouseleave' |
```

Comment:
Maybe `mouseleave `should also be added to `EVENT_TYPES` map?

https://github.com/chartjs/Chart.js/blob/12333b02a5050c8f123826485e32eff7bf0f2d3b/src/platform/platform.dom.js#L24

Then there are quite a few that could still be added, like mousewheel, mouseover, auxclick, touchcancel, focusin, focusout, focus, blur etc.

Also could just use the same thing `addEventLisntener` accepts, which is `keyof HTMLElementEventMap`

Reply:
> Also could just use the same thing addEventLisntener accepts, which is keyof HTMLElementEventMap

This makes a lot of sense, it would solve the issue for good. I'll update the PR.